mypy/stubgenc.py

```diff
@@ -18,6 +18,19 @@
 )
 
 
+_DEFAULT_TYPING_IMPORTS = (
+    'Any'
+    'Dict',
+    'Iterable',
+    'Iterator',
+    'List',
+    'Optional',
+    'Tuple',
+    'Union',
+)
+"""Members of the typing module to consider for importing by default."""
```

Comment:
Style nit: We don't use docstrings for variables. Please instead move this to a comment before the definition.